### DISCUSSION TITLE
Ask Questions here

### DISCUSSION THREAD

**Discussion by TaserTheFox** [TaserTheFox/EsoFur-Interpreter#15] [General]
Ask questions, either about the repo itself, or clarification on the language's wiki page or syntax in general.

**Comment by RichardKanshen** [comment]
Quick Question, I noticed the wiki does not have any missing "OwO What's This?" error? Is that not supposed to throw an error and just execute every line before QwQ, or should it throw some sort of error?

**Reply by TaserTheFox** [reply]
I believe that means the program just doesnt run, and nothing happens

**Reply by RichardKanshen** [reply]
Interesting...
